GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/ransomNote/MagazineNoteCheck.java

```diff
@@ -0,0 +1,39 @@
+package hackkerank.day3.ransomNote;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MagazineNoteCheck {
+    public static void checkMagazine(List<String> magazine, List<String> note) {
+        // Write your code here
+        Map<String, Integer> magazineWords = new HashMap<>();
+        for (String word:magazine){
+
+            if(magazineWords.containsKey(word)){
+                int count = magazineWords.get(word);
+            }else{
+                magazineWords.put(word, 1);
+            }
+        }
+
+        for (String word:note){
```

Comment:
I see missing whitespaces in almost all your code.... please refer to:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Reply:
you need an space before each '{'
you need an space between ':'

In your others classes you did it right!!!!!